strongbox-cron/strongbox-cron-tasks/src/main/java/org/carlspring/strongbox/cron/jobs/CleanupExpiredArtifactsFromProxyRepositoriesCronJob.java

```diff
@@ -57,8 +57,8 @@ public void executeTask(final CronTaskConfigurationDto config)
             }
             catch (NumberFormatException ex)
             {
-                logger.error("Invalid Long value [" + minSizeInBytesText +
-                             "] of 'minSizeInBytes' property. Cron job won't be fired.", ex);
+                logger.error("Invalid Long value [{}] of 'minSizeInBytes' property. Cron job won't be fired.",
```

Comment:
No need for capitalized word here:
```suggestion
                logger.error("Invalid long value [{}] of 'minSizeInBytes' property. Cron job won't be fired.",
```